gridstatus/decorators.py

```diff
@@ -145,6 +145,9 @@ def wrapped_f(*args, **kwargs):
                 elif frequency == "HOUR_START":
                     frequency = pd.DateOffset(hours=1)
 
+                elif frequency == "YEAR_START":
```

Comment:
This was added to support the load for IESO, which is available in CSV files by year.